Tests/Mock/MockPlayerPlugin.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MockPlayerPlugin.swift
+//  ModernAVPlayer_Tests
+//
+//  Created by ankierman on 02/01/2019.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import Foundation
+@testable import ModernAVPlayer
+
+/*
+ * Failed to compile SwiftyMocky protocol with optional method parameter
+ * so I created manually this mock
+*/
+
+final class MockPlayerPlugin: PlayerPlugin {
+
+    private(set) var didPausedCallCount = 0
+    var didPausedLastMediaParam: PlayerMedia?
+    var didPausedLastPositionParam: Double?
+    func didPaused(media: PlayerMedia?, position: Double) {
+        didPausedCallCount += 1
+        didPausedLastMediaParam = media
+        didPausedLastPositionParam = position
+    }
```

Comment:
why don't you use some tuples for keeping last parameters references? <br/>It would be more concise, and ensure you won't miss a potential new parameter
```swift
private(set) var didPausedLastParameters: (playerMedia: PlayerMedia?, position: Double?)
func didPaused(media: PlayerMedia?, position: Double) {
    didPausedCallsCount += 1
    didPausedLastParameters = (playerMedia: media, position: position)
}
```
wdyt?

Reply:
yep, nice !